src/main/java/hw36/ShoppingCartManager.java

```diff
@@ -0,0 +1,96 @@
+package hw36;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+@Component
+public class ShoppingCartManager {
+    private static final Logger LOGGER = Logger.getLogger(ShoppingCartManager.class.getName());
+
+    @Autowired
+    public ProductRepository productRepository;
+
+    @Autowired
+    public Cart cart;
+
+    public void showAllProducts() {
+        LOGGER.info("Available Products:");
+        List<Product> products = productRepository.getAllProducts();
+        for (Product product : products) {
+            LOGGER.info("ID: " + product.getId() + ", Name: " + product.getName() + ", Price: $" + product.getPrice());
+        }
+    }
+
+    public void addToCart(int productId) {
+        Product product = productRepository.getProductById(productId);
+        if (product != null) {
+            cart.addProduct(product);
+            LOGGER.info("Product added to cart: " + product.getName());
+        } else {
+            LOGGER.info("Product not found with ID: " + productId);
+        }
+    }
+
+    public void removeFromCart(int productId) {
+        Product product = productRepository.getProductById(productId);
+        if (product != null) {
+            cart.removeProduct(product);
+            LOGGER.info("Product removed from cart: " + product.getName());
+        } else {
+            LOGGER.info("Product not found with ID: " + productId);
+        }
+    }
+
+    public void showCartItems() {
+        List<Product> cartItems = cart.getCartItems();
+        if (cartItems.isEmpty()) {
+            LOGGER.info("The cart is empty.");
+        } else {
+            LOGGER.info("Cart Items:");
+            for (Product product : cartItems) {
+                LOGGER.info("ID: " + product.getId() + ", Name: " + product.getName() + ", Price: $" + product.getPrice());
+            }
+        }
+    }
+
+    public static void main(String[] args) {
+        AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext(AppConfig.class);
+        ShoppingCartManager manager = context.getBean(ShoppingCartManager.class);
+        Scanner scanner = new Scanner(System.in);
```

Comment:
Sorry, you're not allowed to use scanner, use IO instead, BufferedReader or any other reader of your choice